handbook/handbook.in

```diff
@@ -287,7 +287,7 @@ programs are automatically executed when the graphical interface starts.
 Further, it allows you to enable/disable some components of the NomadBSD
 desktop. The changes take place after logging out and in again.</p>
 
-<p><img src="images/dsbautostart-screenshot.png" alt="" />
+<p><img src="handbook/images/dsbautostart-screenshot.png" alt="" />
```

Comment:
Same here.

Reply:
Thanks for the great feedback! Fixed :-)